std/wasi/snapshot_preview1_test_runner.ts

```diff
@@ -0,0 +1,20 @@
+import Context from "./snapshot_preview1.ts";
```

Comment:
Why not just std/wasi/test_runner ?

Reply:
It's bit of a mouthful, am just going by the snapshot (ABI version) being used which in this case is snapshot_preview1.